mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py

```diff
@@ -58,4 +60,46 @@ def get_email_campaigns(self, **kwargs):
         connection = self.handler.connect()
         email_campaigns_api_instance = sib_api_v3_sdk.EmailCampaignsApi(connection)
         email_campaigns = email_campaigns_api_instance.get_email_campaigns(**kwargs)
-        return [email_campaign for  email_campaign in email_campaigns.campaigns]
\ No newline at end of file
+        return [email_campaign for  email_campaign in email_campaigns.campaigns]
+    
+    
+    def update(self, query: 'ast.Update') -> None:
+        """
+        Updates data in Sendinblue "PUT /emailCampaigns/{campaignId}" API endpoint.
+        
+        Parameters
+        ----------
+        query : ast.Update
+            Given SQL UPDATE query
+            
+        Returns
+        -------
+        None
+        
+        Raises
+        ------
+        ValueError
+            If the query contains an unsupported condition
+        """
+        update_statement_parser = UPDATEQueryParser(query)
+        values_to_update, where_conditions = update_statement_parser.parse_query()
+
+        email_campaigns_df = pd.json_normalize(self.get_email_campaigns())
+        update_query_executor = UPDATEQueryExecutor(
+            email_campaigns_df,
+            where_conditions
+        )
+
+        email_campaigns_df = update_query_executor.execute_query()
+        campaign_ids = email_campaigns_df['id'].tolist()
+        self.update_email_campaigns(campaign_ids, values_to_update)
+
+    def update_email_campaigns(self, campaign_ids: List[int], values_to_update: Dict) -> None:
+        connection = self.handler.connect()
+        email_campaigns_api_instance = sib_api_v3_sdk.EmailCampaignsApi(connection)
+
+        for campaign_id in campaign_ids:
+            try:
+                email_campaigns_api_instance.update_email_campaign(campaign_id, values_to_update)
+            except ApiException as e:
+                print(f"Exception when updating campaign {campaign_id}: {e}")
```

Comment:
Lets log and raise the exception instead of printing

Reply:
okay

Reply:
@ZoranPandovski  made the changes  please review it